.github/actions/plan/modes.ts

```diff
@@ -4,6 +4,8 @@ export type Mode = {
   cargoArgs: string;
   cargoCacheKey: string;
   platformIndependent?: true;
+  artifactSelector?: "peer" | "runtime";
+  artifactFeaturesMarker?: "evm-tracing";
```

Comment:
```suggestion
  artifactMarker?: string;
```

More generic - doesn't have to be specifically about the features, and can easily be any string

Reply:
Don't see the type change to `string`